samples/KHB/KHB.cfg

```diff
@@ -0,0 +1,153 @@
+
+
+
+# Simulation type, corresponding to a specific type of setup and dedicated class defined under projects/ in the source
+project = KHB
+
+# Let the simulation adapt the time step according to the CFL condition
+dynamic_timestep = 1
+
+# Arbitrary name given to the simulated ion population. This project does not support multiple concurrent ion populations.
+ParticlePopulations = proton
+
+# Specifications of the ion population
+[proton_properties]
+mass = 1
+mass_units = PROTON
+charge = 1 
+
+# Field solver options
+[fieldsolver]
+ohmHallTerm = 2
+maxSubcycles = 50
+
+# Set of options shared by all projects
+[Project_common]
+# Seed used by a random number generator for setting random phases for the initial perturbation
+seed = 124
+
+# Set of options specifying conditions under which the run will exit gracefully to avoid potential crashing
+[bailout] 
+max_memory = 250 # set based on system
+
+# Input/Output
+[io]
+diagnostic_write_interval = 10
+write_initial_state = 0
+#restart_walltime_interval = 9999999.0
+#number_of_restarts = 1
+
+system_write_t_interval = 5.0
+system_write_file_name = bulk
+
+# With these set to zero, no velocity distribution functions are saved in the bulk file 
+system_write_distribution_stride = 0
+system_write_distribution_xline_stride = 0
+system_write_distribution_yline_stride = 0
+system_write_distribution_zline_stride = 0
+
+# Spatial grid and run completion parameters
+[gridbuilder]
+x_length = 42
+y_length = 28
+z_length = 1
+
+x_min = -10500.0E+3
+x_max = +10500.0E+3
+y_min = -7000.0E+3
+y_max = +7000.0E+3
+z_min = -250.0E+3
+z_max = +250.0E+3
+
+timestep_max = 10000
+t_max = 350.0
+
+# Velocity space parameters for the ion population declared at the top
+[proton_vspace]
+vx_min = -3000.0E+3
+vx_max = +3000.0E+3
+vy_min = -3000.0E+3
+vy_max = +3000.0E+3
+vz_min = -3000.0E+3
+vz_max = +3000.0E+3
+vx_length = 30
+vy_length = 30
+vz_length = 30
+
+# Velocity sparsity parameters for the ion population declared at the top
+[proton_sparse]
+minValue = 1.0E-15
+
+# Simulation boundary conditions; periodicity and declaration of boundary types that will be required
+[boundaries]
+periodic_x = no
+periodic_y = yes
+periodic_z = yes
+boundary = Maxwellian
+
+# Uniform Maxwellian distribution on set faces of the simulation domain
+[maxwellian]
+face = x-
+face = x+
+
+# Files containing the Maxwellian properties at the set faces of the simulation domain
+[proton_maxwellian]
+dynamic = 0
+file_x- = mxm.dat
+file_x+ = mxp.dat
+
+# Variables (data reduction operators) to be written to the output VLSV files and to the diagnostic.txt file
+# fg_* variables on the finest-resolution field solver grid (/!\ can be several GB per component in large runs)
+# vg_* variables on the adaptive mesh storing the plasma distributions and moments
+# populations_* variables are stored separately for each specified ion population as <population>/vg_*
+[variables]
+output = populations_vg_rho
+output = populations_vg_v
```

Comment:
I suggest that you add a few more parameters relevant for plotting/analysing KH as an example here.